docs/api.md

```diff
@@ -3607,7 +3607,7 @@ ResourceType will be one of the following: `document`, `stylesheet`, `image`, `m
 - returns: <[Frame]> A [Frame] that initiated this response.
 
 #### response.headers()
-- returns: <[Object]> An object with HTTP headers associated with the response. All header names are lower-case.
+- returns: <[Object]<[string], [string]>> An object with HTTP headers associated with the response. All header names are lower-case.
```

Comment:
I didn't know we support these fancy types! Joel's magic?

Reply:
Not sure if you do, but I do in Python and you should do too!